src/date-picker/date-picker.scss

```diff
@@ -4,16 +4,23 @@
 @import '../icon/index';
 @import '../menu/index';
 
+$iui-date-picker-cell-width: 40px;
```

Comment:
Could this be 36px as well or does this introduce issues?

Reply:
Well yeah, Thats the minimum width I can give to the day, so everything can fit in nicely :)
But try it out, maybe you will find out better solution

Reply:
Changing it to 36px doesn't seem to cause any issues.  I think we can safely change this to 36px.